satellite/satellitedb/database.go

```diff
@@ -110,8 +112,8 @@ func (db *DB) Attribution() attribution.DB {
 }
 
 // OverlayCache is a getter for overlay cache repository
-func (db *DB) OverlayCache() overlay.DB {
-	return &overlaycache{db: db.db}
+func (db *DB) OverlayCache(isUpDur time.Duration) overlay.DB {
```

Comment:
Won't this end up creating multiple caches?

Reply:
Yes, I'm fixing that now. Good catch, I wondered if anyone would notice before I had a chance to fix it and change how it works. :)

Reply:
This is now fixed.